packages/vscode-extension/src/commands/ImportFiles.ts

```diff
@@ -0,0 +1,139 @@
+	}
+
+	protected init(fsPath?: string) {
+		super.init(fsPath);
+		this.importFileService.setVsConsoleLogger(this.vsConsoleLogger);
+		this.listFilesService.setVsConsoleLogger(this.vsConsoleLogger);
+	}
+
+	protected async execute() {
+		if (!this.activeFile) {
+			// Already checked in validate
+			return;
+		}
+
+		const fileName = path.basename(this.activeFile, '.xml');
+
+		const selectedFilesPaths: string[] | undefined = await this.getSelectedFiles();
+		if (!selectedFilesPaths) {
+			return;
+		}
+
+		const excludeProperties = await window.showQuickPick(
+			[this.translationService.getMessage(ANSWERS.YES), this.translationService.getMessage(ANSWERS.NO)],
+			{
+				placeHolder: this.translationService.getMessage(IMPORT_FILES.QUESTIONS.EXCLUDE_PROPERTIES, fileName),
+				canPickMany: false,
+			}
+		);
+
+		const override = await window.showQuickPick(
+			[this.translationService.getMessage(ANSWERS.YES), this.translationService.getMessage(ANSWERS.NO)],
+			{
+				placeHolder:
+					selectedFilesPaths.length > 1
+						? this.translationService.getMessage(IMPORT_FILES.QUESTIONS.OVERRIDE)
+						: this.translationService.getMessage(IMPORT_FILES.QUESTIONS.OVERRIDE_SINGLE, selectedFilesPaths[0]),
+				canPickMany: false,
+			}
+		);
+
+		if (!override || override === this.translationService.getMessage(ANSWERS.NO)) {
+			this.messageService.showInformationMessage(this.translationService.getMessage(IMPORT_FILES.PROCESS_CANCELED));
+			return;
+		}
+
+		const destinationFolder = this.executionPath
+			? path
+					.dirname(this.activeFile)
+					.split(this.executionPath + '\\src')[1]
+					.replace('\\', '/')
+			: path.dirname(this.activeFile);
+
+		const statusBarMessage = this.translationService.getMessage(IMPORT_FILES.IMPORTING_FILE);
+		const actionResult = await this.importFileService.importFiles(
+			selectedFilesPaths,
+			destinationFolder,
+			statusBarMessage,
+			this.executionPath,
+			excludeProperties === this.translationService.getMessage(ANSWERS.YES)
+		);
+
+		this.showOutput(actionResult);
+	}
+
+	private async getSelectedFiles(): Promise<string[] | undefined> {
+		if (!this.isFileSelected) {
+			const fileCabinetFolders: FolderItem[] = await this.listFilesService.getListFolders();
+			const selectedFolder: QuickPickItem | undefined = await this.listFilesService.selectFolder(fileCabinetFolders);
+			if (!selectedFolder) {
+				return;
+			}
+			const files = await this.listFilesService.listFiles(selectedFolder.label);
+			const selectedFiles: QuickPickItem[] | undefined = await this.listFilesService.selectFiles(files);
+
+			if (!selectedFiles) {
+				this.messageService.showInformationMessage(this.translationService.getMessage(IMPORT_FILES.PROCESS_CANCELED));
+				return;
+			}
+			return selectedFiles.map((file) => file.label.replace(/\\/g, '/'));
+		} else {
+			if (this.activeFile) {
+				const filePath = this.executionPath
+					? this.activeFile.split(this.executionPath + '\\src\\FileCabinet')[1].replace(/\\/g, '/')
```

Comment:
Similar to the previous comment use `this.getProjectFolderPath()`  `FileCabinet` is already on node-cli ApplicationConstants, which is also imported through ExtensionUtils.ts.

Reply:
Fixed